README.md

```diff
@@ -28,6 +29,14 @@ provider, you'll need to install a new provider plugin.
 Additional information about other dependencies required by vagrant-libvirt are
 available at https://github.com/vagrant-libvirt/vagrant-libvirt
 
+## Docker
+
+ansible-cira also supports docker container instead of OpenStack and Vagrant.
```

Comment:
In addition to supporting OpenStack and Vagrant deployments, ansible-cira can also utilize Docker containers.